main.py

```diff
@@ -1,10 +1,52 @@
-import init_django_orm  # noqa: F401
+import init_django_orm  # noqa
 
-from db.models import Race, Skill, Player, Guild
+import json
+from db.models import Race, Skill, Guild, Player
+
+
+def create_race(data: dict) -> Race:
```

Comment:
Your function name is ambiguous since it is not supposed to create skills. Rename it to `create_race_with_skills` for proper encapsulation.